tracker/tests.py

```diff
@@ -1,3 +1,65 @@
-from django.test import TestCase
-
+from django.test import TestCase, Client
+from django.urls import reverse, resolve
+import json
+from .models import *
+from .views import *
+import datetime
 # Create your tests here.
+
+class TestModels(TestCase):
+	"""docstring for ClassName"""
+	def setUp(self):
+		Task.objects.create(
+			name = 'xx',
+			planning = '07:04',
+			estimate = '02:50',
+			realtime = '04:52',
+			risk = 'OK',
+			level = 'easy',
+			notes = 'azerty',
+			estimateb_by_calc = '4:27',
+			correctness = 50,
+		)
+
+	def test_Task(self):
+		self.new = Task.objects.get(risk = 'OK',)
+		self.assertEquals(self.new.realtime, datetime.time(4, 52))
+
+
+class TestUrls(TestCase):
+	"""docstring for ClassName"""
+	def test_urls(self):
```

Comment:
Could your rename the name of the test to be more explicit about what it is testing?